packages/internal/src/types.js

```diff
@@ -0,0 +1,5 @@
+// @ts-check
```

Comment:
What is the purpose of this file? Just a placeholder for future exported types not defined elsewhere? In that case it should be re-exported from the index